admin_guide/high_availability.adoc

```diff
@@ -404,38 +413,36 @@ displayed in decimal (`493`).
 [[vrrp-preemption]]
 === VRRP Preemption
 
-When a host leaves the *FAULT* state by passing the check script it can become *MASTER*.
-If the new host has lower priority than the current *MASTER* it becomes a *BACKUP*. If
-it has a higher priority the preemption strategy determines what to do. The *nopreempt*
-strategy does not move *MASTER* from the lower priority host to the higher priority host.
-With *preempt 300*, the default, *keepalived* waits the specified 300 seconds and moves
-*MASTER* to the higher priority host.
+When a host leaves the *FAULT* state by passing the check script, the host
+becomes a *BACKUP* if the new host has lower priority than the current *MASTER*.
+However, if it has a higher priority, the preemption strategy determines it's
+role in the cluster. 
```

Comment:
Same here with BACKUP and MASTER. I see where there are apparently MASTER and BACKUP states; but here, MASTER is not referring to the state, but the "type" of system, right?